src/web/assets/cp/src/js/Uploader.js

```diff
@@ -128,6 +124,7 @@ Craft.Uploader = Craft.BaseUploader.extend(
       autoUpload: false,
       sequentialUploads: true,
       maxFileSize: Craft.maxUploadSize,
+      replaceFileInput: false,
```

Comment:
This creates duplicate inputs, which causes problems and is only needed for legacy iframe transport uploading.